packages/components/src/components/linear-progress/linear-progress-container.jsx

```diff
@@ -3,16 +3,18 @@ import PropTypes from 'prop-types';
 import { LinearProgress } from './linear-progress.jsx';
 
 const LinearProgressContainer = ({ timeout, action, render, className }) => {
-    const [timeout_state, setTimeout] = React.useState(timeout / 1000);
-    const [current_tick, setCurrentTick] = React.useState(Math.round(timeout / 1000));
-    const [total_ticks] = React.useState(Math.round(timeout / 1000));
+    const get_popup_timer = sessionStorage.getItem('popup_timer');
+    if (!get_popup_timer) sessionStorage.setItem('popup_timer', timeout / 1000);
+    const popup_timeout = !get_popup_timer ? timeout / 1000 : get_popup_timer;
 
-    const getProgress = () => 100 - Math.round((current_tick / total_ticks) * 100);
+    const [timeout_state, setTimeout] = React.useState(popup_timeout);
```

Comment:
Can we call the method `setTimeoutState` for consistency?

Reply:
Fixed